### DISCUSSION TITLE
simbot怎么导入Mirai插件

### DISCUSSION THREAD

**Discussion by ConstantineQAQ** [ForteScarlet/simpler-robot#426] [不要在这里发帖子了]
在Mirai官网上看到了一些有意思的插件，想导入到simbot里，但是不是很会

**Comment by ForteScarlet** [comment]
simbot的mirai组件直接使用 `mirai-core` ，而插件通常需要在 `mirai-console` 环境下使用。目前的mirai组件不支持使用 `mirai-console` 插件。
